openwisp_controller/config/tests/test_device.py

```diff
@@ -64,3 +64,10 @@ def test_device_hardwareid_unique_together(self):
         }
         with self.assertRaises(ValidationError):
             self._create_device(**kwargs)
+
+    def test_config_device_without_org(self):
+        device = Device()
+        try:
+            self._create_config(device=device)
+        except Exception as e:
```

Comment:
catch only `ValidationError` from `django.core.exceptions`

Reply:
Ok. But the exception that is thrown is the RelatedObjectDoesNotExist exception. don't you think catching the ValidationError exception can let go the RelatedObejctDoesNotExist exception?

Reply:
@nemesisdesign , don't you think that catching the exception should be as specific as possible? 
Like in this case wouldn't it better to catch the corresponding `DoesNotExist` exception? 
@NoumbissiValere , it will not be wise to catch `Exception` because then any exception will trigger that, and that's not something you'd want because you want handle various exceptions differently.